src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -169,8 +169,23 @@ function generateChartDescription(
   projection: Projection,
   noInterventionProjection: Projection,
 ) {
-  // TODO(sgoldblatt): figure out how to get people number data from projection
+  const noInterventionDate = noInterventionProjection.dateOverwhelmed;
+  const reopeningText = noInterventionDate
+    ? `happen in a slow and phased fashion. If all restrictions were completely lifted today, ` +
+      `hospitals are projected to become overloaded by ${formatDate(
+        noInterventionDate,
+      )}.`
+    : `happen in a slow and phased fashion.`;
+
+  if (!projection.isInferred) {
+    return (
+      `We don't have enough data to project whether hospitals are likely to become overloaded. ` +
+      `Reopening should always ${reopeningText}`
```

Comment:
i guess it's maybe a lil confusing to me to say we don't have enough data to project whether hospitals are overloaded and then say hospitals are projected to become overloaded?

Reply:
What do you suggest? 😁

Basically we don't have enough data to indicate how well in-place interventions are working, so we can't make a projection.  But we do know that if all restrictions are lifted, we'll return to a r-value of ~3 and things will go poorly. So we can project a hospital overload date for that.